Update the command line tutorial with the options currently in `master` (#3973)
godotengine/godot-docs: getting_started/editor/command_line_tutorial.rst

To export a debug version of the game, use the ``--export-debug`` switch
instead of ``--export``. Their parameters and usage are the same.

> To export only a PCK file, use the ``--export-pack`` switch followed by the
> preset name and output path (with the file extension) instead of ``--export``.
> The output path extension determines whether it will be in PCK or ZIP format.

Reviewer: ```suggestion
To export only a PCK file, use the ``--export-pack`` option followed by the 
preset name and output path, with the file extension, instead of ``--export``.
The output path extension determines the package's format, either PCK or ZIP.
```
